README.md

```diff
@@ -80,7 +87,7 @@ $ simple-ec2 launch -h
 Launch an Amazon EC2 instance with the default configurations. All configurations can be overridden by configurations provided by configuration files or user input.
 
 Usage:
-  simple-ec2 launch [flags]
+  simple-ec2 launch|version [flags]
```

Comment:
Can you remove `|version` now that it has its own section?

Reply:
Pushed the change